components/suggestion/search_channel_suggestion/search_channel_suggestion.tsx

```diff
@@ -70,7 +68,9 @@ function itemToName(item, currentUser) {
 }
 
 export default class SearchChannelSuggestion extends Suggestion {
-    render() {
+    private node = React.createRef<HTMLDivElement>();
```

Comment:
This should be done in the constructor, not in the declared field.
Also if you're going to use this pattern for refs (which is the better pattern), anywhere the ref is used needs to be updated as well. Although I don't see any references to `node`, so maybe we don't even need the ref?

Reply:
Yes, I already remove it.